cleanlab/multiannotator.py

```diff
@@ -424,6 +434,14 @@ def get_majority_vote_label(
         for idx, label_mode in tied_idx.items():
             majority_vote_label[idx] = np.random.choice(label_mode)
 
+    unique_ma_labels = np.unique(labels_multiannotator.replace({pd.NA: np.NaN}).astype(float))
+    unique_ma_labels = unique_ma_labels[~np.isnan(unique_ma_labels)]
+
+    if len(set(majority_vote_label).symmetric_difference(set(unique_ma_labels))) != 0:
+        print(
+            "CAUTION: Number of unique classes has been reduced from the original data, likely due to some classes being rarely annotated"
```

Comment:
```suggestion
            "CAUTION: Number of unique classes has been reduced from the original data when establishing consensus labels, likely due to some classes being rarely annotated. If training a classifier on these consensus labels, it will never see any of the omitted classes unless you manually replace some of the consensus labels."
```